packs/data/other-effects.db/effect-dazzled-until-end-of-your-next-turn.json

```diff
@@ -0,0 +1,47 @@
+{
+    "_id": "uwYGEeqvt8pwjhXa",
+    "img": "systems/pf2e/icons/effects/critical-effect.webp",
+    "name": "Effect: Dazzled until end of your next turn",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #51]{Critical Fumble Deck #51}, @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #52]{Critical Fumble Deck #52}</strong></p>\n<p data-pm-slice=\"1 1 [&quot;section&quot;,{&quot;_preserve&quot;:{&quot;class&quot;:&quot;fumble-deck&quot;}},&quot;blockquote&quot;,{&quot;_preserve&quot;:{}}]\">You are <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.conditionitems.TkIyaNPgTZFBCCuh\" data-id=\"TkIyaNPgTZFBCCuh\" data-type=\"Item\" data-pack=\"pf2e.conditionitems\" data-tooltip=\"Condition Item\" data-args=\"\">Dazzled</a> until the end of your next turn.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Effect applied by @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #51]{Critical Fumble Deck #51}, @UUID[Compendium.pf2e.criticaldeck.Critical Fumble Deck #52]{Critical Fumble Deck #52}</strong></p>\n<p data-pm-slice=\"1 1 [&quot;section&quot;,{&quot;_preserve&quot;:{&quot;class&quot;:&quot;fumble-deck&quot;}},&quot;blockquote&quot;,{&quot;_preserve&quot;:{}}]\">You are <a class=\"content-link\" draggable=\"true\" data-uuid=\"Compendium.pf2e.conditionitems.TkIyaNPgTZFBCCuh\" data-id=\"TkIyaNPgTZFBCCuh\" data-type=\"Item\" data-pack=\"pf2e.conditionitems\" data-tooltip=\"Condition Item\" data-args=\"\">Dazzled</a> until the end of your next turn.</p>"
```

... wut

Reply:
Wonder what happened there? Should be fixed now.